pkg/audit/manager.go

```diff
@@ -1061,19 +1064,33 @@ func emitEvent(constraint *unstructured.Unstructured,
 		logging.ResourceNamespace:    rnamespace,
 		logging.ResourceName:         rname,
 	}
+
 	reason := "AuditViolation"
-	ref := getViolationRef(gkNamespace, resourceGroupVersionKind.Kind, rname, rnamespace, constraint.GetKind(), constraint.GetName(), constraint.GetNamespace())
+	enamespace := gkNamespace
+	if *auditEventsInvolvedNamespace && len(rnamespace) > 0 {
+		enamespace = rnamespace
```

Comment:
Perhaps add a note in the readme and docs that cluster scoped resources would create events in the GK namespace.